File: algorithms/easy/runningSum.py
def runningSum(arr):
  sum = 0 
  result = []
  for num in arr:
      sum += num
      result.append(sum)   
  return result



print(runningSum([1,2,3]))
print(runningSum([1,1,1]))
print(runningSum([30,10,20]))

File: algorithms/easy/runningSum.js
const runningSum = (arr) => {
  let sum = 0;
  let results = [];
  for (let num of arr) { 
    sum += num;
    results.push(sum)
  }
  return results;
}

console.log(runningSum([1,2,3]))
console.log(runningSum([1,1,1]))
console.log(runningSum([30,1,1]))